tsconfig.json

```diff
@@ -0,0 +1,37 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "baseUrl": "./",
+    "paths": {
+      "components/*": ["src/components/*"],
+      "layouts/*": ["src/layouts/*"],
+      "lib/*": ["src/lib/*"],
+      "pages/*": ["src/pages/*"],
+    },
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "noImplicitAny": true,
+    "forceConsistentCasingInFileNames": true,
+    "suppressImplicitAnyIndexErrors": true,
```

Comment:
This is not recommended by typescript, can we add a comment for when it can be removed?

Reply:
I added a task here: https://github.com/orgs/uselagoon/projects/4/views/1?pane=issue&itemId=23629169
I'll have a separate PR for some of these mentioned rules with required changes